src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -2319,6 +2323,9 @@ static int __icap_download_bitstream_axlf(struct platform_device *pdev,
 			xocl_subdev_create(xdev, &subdevs[num_dev].info);
 	}
 
+	if (ICAP_PRIVILEGED(icap))
```

Comment:
why we don't have if (ICAP_PRIVILEGED(icap)) when we call xocl_cmc_release_access() above?

Reply:
The code above handles mgmt and user both, line 2327 depends on loaded new fdt line (2320 - 2324). So a minimum change is to add if(ICAP_PRIV...).

Reply:
Not sure I'm following...but we don't download xclbin on user side and we don't have CMC on user side. So, what we need to handle from user side?